cmd/server/cmd/BUILD

```diff
@@ -8,6 +8,7 @@ go_library(
         "inventory.go",
         "root.go",
         "server.go",
+        "version.go",
```

Comment:
I've now moved the version command into a shared package (along with OutFn and ErrorFn).